index.html

```diff
@@ -108,4 +109,11 @@ <h2 class="d-md-none fw-bold">Your direct access to Noroff's best candidates</h2
   <footer></footer>
 </body>
 
-</html>
\ No newline at end of file
+</html>
+=======
+      </section>
+    </main>
+    <footer></footer>
+  </body>
+</html>
+>>>>>>> Stashed changes
```

Comment:
You seem to have some stashed changes leftover in your code that needs to be removed